src/components/Tabs/TabList.tsx

```diff
@@ -27,18 +34,20 @@ export interface TabListProps {
   className?: string;
 }
 
-export const TabList: React.FC<TabListProps> = ({
+export const TabList: React.FC<TabListProps & Partial<TabListOptions>> = ({
```

Comment:
Thought passing the types of the base component for these would be nice. Any concerns with this?

Reply:
I _think_ it'd be okay, my only worry may be there could be something we don't want to support from a design perspective (e.g., `orientation`, etc.).

We had initially only exposed the props we approved from a design perspective intentionally ( cc @dexterca ) .  I'm a bit torn personally on exposing this, my gut feeling says leave things as-is for now until we have a case where we specifically want one of these additional props, but I don't have strong opinions either way 😄

Reply:
Sounds good, lets see if there are any other opinions else we can hold off